tests/boardfarm_plugins/boardfarm_prplmesh/devices/prplmesh_docker.py

```diff
@@ -33,7 +33,8 @@ def __init__(self, *args, **kwargs):
         # Getting unic ID to distinguish devices and network they belong to
         self.unique_id = os.getenv("SUDO_USER", os.getenv("USER", ""))
 
-        self.name = "-".join((config.get("name", "prplmesh_docker"), self.unique_id))
+        self.name = config.get("name", "prplmesh_docker")
```

Comment:
Is self.name something that is used by boardfarm?

If not, I think it would be better to call this something else, and to keep using self.name for the thing with unique_id appended. Otherwise it's confusing that PrplMeshDocker.name of "prplmesh_docker", but the corresponding ALEntity.name is "prplmesh_docker-arnout".

Reply:
Yep, it is. `self.name` should be one of boardfarm build-in [enum](https://github.com/mattsm/boardfarm/blob/100521fde1fb67536682cafecc2f91a6e2e8a6f8/boardfarm/lib/DeviceManager.py#L25). If it doesn't match - we can't address devices directly by name in test.

Reply:
In that case, we should probably (eventually) rename ALEntity.name.